conn/gpio/gpiotest/gpiotest.go

```diff
@@ -128,5 +129,60 @@ func (p *PinPWM) PWM(duty gpio.Duty, period time.Duration) error {
 	return nil
 }
 
+// since returns time in µs since the test start.
```

Comment:
In general, the log.Print() statement already include time. The reason the gpiosmoketest prints time is for specialized function which I don't think is generalizeable. As such, I don't think the smoke test should use gpiotest.LogPinIO.